rust-webvr-api/src/vr_display.rs

```diff
@@ -45,6 +46,14 @@ pub trait VRDisplay: Send + Sync {
     /// Must be called in the render thread
     fn submit_frame(&mut self);
 
+    /// Renders a VRLayer from an external texture, and submits it to the device.
+    /// Must be called in the render thread
+    #[allow(unused_variables)]
+    fn submit_layer(&mut self, gl: &Gl, layer: &VRLayer) {
+        self.render_layer(layer);
```

Comment:
Should we remove the old `render_layer` function? It may be confusing to have both in the API. We could add an array of layers if we need more than one in the future

Reply:
I was planning on marking the old API as "deprecated" and removing it in the next breaking-changes-version.

Reply:
Perfect, update the PR with deprecated for the old method and let's merge this